### PR TITLE
[TensorV2] apply() function to apply a given function

### PR DESCRIPTION
**Pull request by djeong20**
This pull request implements a new function called apply(), which applies a given function element-by-element to a tensor.
The resulting tensor has the same shape as the input tensor, but each element has been transformed by the given function.

**Self-evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test:   [X]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

**Review by skykongkong8 — APPROVED** [review]

**Review by Hyoungsukim — APPROVED** [review]
lgtm

**Review by baek2sm — APPROVED** [review]
LGTM

**Review by SeoHyungjun — APPROVED** [review]
LGTM!

**Review by DonghakPark — APPROVED** [review]
LGTM!

**Review by lhs8928 — APPROVED** [review]
LGTM except minor comment

File: nntrainer/tensor/tensor_v2.cpp [hunk 1]
```diff
@@ -154,6 +154,32 @@ TensorDim::Format TensorV2::getFormat() const { return itensor->getFormat(); }
...
+  return itensor->getContiguous();
+}
+
+const std::array<size_t, TensorDim::MAXDIM>
+TensorV2::getStrides() const noexcept {
+  return itensor->getStrides();
+}
+
+bool TensorV2::checkContinuous(unsigned int np1, unsigned int np2) const {
+  if (getFormat() == Tformat::NCHW) {
```
[the remark this answers is not in the document]
Comment (djeong20) [reply nntrainer/tensor/tensor_v2.cpp:181]: good point. fixed!

**Review by jijoongmoon — APPROVED** [review]
LGTM

**Pull request merged**